alembic/versions/3f4e119594fe_extend_confluence_url_length.py

```diff
@@ -0,0 +1,56 @@
+# type: ignore
+# flake8: noqa
+"""extend confluence_url length
+
+Revision ID: 3f4e119594fe
+Revises: e6dc052f0caf
+Create Date: 2024-04-02 11:01:50.116786
+
+"""
+
+import logging
+
+import sqlalchemy as sa
+
```

Comment:
I think this empty line may be deleted. Are you using `isort`?

Reply:
We are using `isor` here indeed. I just tried locally making a change so Isort claims about it, and on its formatting it sets the imports as they are now. So I think we are good :+1: (I'm using `ts-pre-commit-config 0.9.5` btw).